server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
At OT we have a hobknob-private repo then npm install hobknob and then run the following:

```javascript
node node_modules/hobknob/server/app.js
```

This PR will break our current consumption, I think.